src/js/view/controls/rightclick.js

```diff
@@ -50,6 +51,12 @@ export default class RightClick {
                 link: 'http://www.adobe.com/software/flash/about/'
             });
         }
+        if (this.shortcutsTooltip) {
+            const secondToLastIndex = menu.items.length - 1;
```

Comment:
we can improve minification if we make an items const: `const items = menu.items`. We should declare it on line 45, since it can be used as early as line 48. 
We can then get rid of the `secondToLastIndex` var since `items.length - 1` is still pretty readable

Reply:
Done!